packages/datepicker-react/src/CalendarIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+interface Props {
+    className?: string;
+}
+
+export const CalendarIcon = ({ className }: Props) => (
```

Comment:
Use SVGO to optimise this svg. 

See difference in output size:
https://babeljs.io/repl/#?browsers=ie%2011&build=&builtIns=false&spec=false&loose=false&code_lz=MYewdgzgLgBAwgQwDYFMwBMECcCSowwC8MAFAN4zBIIQQByCAtijAL4BcMACliAA4QAlEQB8pAFAwpMADwQAbgHNK1Wg2aEyVGvSYpWMeQEsUAdwBCIAB6EARAAYYjgEwAWGG9swAZkaRI7MHAULytGJEg7AAsoKD52AHoE0xSAOlMAZlSQLEUE53tChIVFWxFJaUqZPgQoKJhoXgBrFDtgAFcsLDQoOBAkHK90OwBZVIBWGCzxqOcM-QKACQmALxGANgmneQB2EYBGHa37XdsE8sqqmrqfPwDbDq6evoGsWwrLz5hh2xHXLYAnBNFusAGr7faLf7jNbQmD7Vwg-T7YHQ2FbfYADmBYKWaJGAJgQJmKPG4MhALWhIRUVJyOBlIJ8OxJImoKWjP20yJwP2ZIhUQAtKS1lyMYi-fSZsLVgduVjeWSCkKRQdMTzWfzIVjReqaXTSYsdWrmRNaWyljqzhdpDJikpyoIANziV34aDwZBoTC4fDOIikCjaNR6NicHj8ISiCRVEoqHTqVpaVS6ZgwAD88ZDac4tlsBmMZksNgcTg87k8t38gWCoXCkVsMTiiWSaUy2Vy-UK9ntpRtnxk3WAsBL9gmXgAnnZpl5TEZ0HU7HMvFEUEZFDEl_YvI0QC02p1umBev1BjBzh9LjIkEYwCwrPs7JtxqFnE_xzAJ2_bDsd1Bmq0DyHs8p5vOe_ZXjed4wA-diHB-VjfvBL6ft-v4NP-e6AY8R4nq8XgXl8shDiOdjQpOdjErO86LrYpIrmuG5QHBH6-NWQFPMeLxnoRXyDigw4wWRH5TnRrjUQuUQsShq7rpudGsXcB6cXhPEQVUJFCbY5GfnBLISbR9EwLJTHSV4bH3DhIH4eBl4aQJpG2ACFFOR-c6SWZxmMfJRkWcpuHcWBvEDppJbObpYkGVJCkyd5zExeZSkcQFoEEeptqhZRLkKih7mGR-Jk-Yp7FWVxqW2UR_GCSWYooaJVEwHl0VGYV8W-UlpWqUF6VSFVjm1dl4mNTRzUFXFnl-cl1lqXZGUOVpA0RTlUWea1E0dcBZU2cFV6ZXRmIuQ1TWreNCVVpZm1dWls29XtWKDStZ1rWdk2dYF12VXdB1Lfpw0eU9p3tSVl3vRVtq9o6LpAA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=true&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env%2Ces2015%2Creact%2Ces2015-loose%2Ctypescript%2Cenv&prettier=false&targets=&version=7.8.4&externalPlugins=

Reply:
Kan gjøres i vs code med ctrl+shift+p `svg minify` med https://marketplace.visualstudio.com/items?itemName=jock.svg

Reply:
Glemmer at React insisterer på å lage funksjoner av hver eneste tag i SVG-er 😑

Reply:
Yup. Det interessante er at den optimaliserte SVGen faktisk er større en den uoptimaliserte SVGen, hvis man hadde sammenlignet SVG mot SVG og ikke laget det med React.

Reply:
Nettop! Og **mye** vanskeligere å lese 😣